### ISSUE TITLE
CHANGELOG for v1.11.1/v1.10.5/v1.9.8

### ISSUE THREAD

**Issue by 3nprob** [hashicorp/vault#16408]
Release notes only mention a single commit each but looking at the commits,  there seem to be more changes included in these releases.

Also, v1.11.0 is still marked as Unreleased.

**Comment by pigcivil** [comment]
[![image](https://user-images.githubusercontent.com/85682049/180358000-cc74ba0e-6d88-415d-bfe1-6b8894e5c0da.jpeg)]()

**Comment by 3nprob** [comment]
v1.11.2 is also released.

**Comment by hsimon-hashicorp** [comment]
Hi there - we only keep the changelog on the main branch up to date. The release branches don't always have up-to-date information. In the future, we'd like to explore automation to make this process more seamless, but we're not quite there yet. Please let me know if this answers your question. Thanks! :)

**Comment by 3nprob** [comment]
Right, at the time this issue was files, the changelog on `main` was not up to date either but it seems to have been resolved some days later.

Automated or not, I think *especially* for a product like Vault, release notes at the time of release (enumerating at least potential breaking changes + security advisories) are crucial. At least I know I freaked out a bit and started enumerating commits to see just how urgent it would be for us to circumvent protocol and push this upgrade out ASAP considering it was urgent enough to skip the release notes.

I don't think I'm the first to react like this given all past releases have a changelog and all other HC products have them bundled with the release itself)
